packages/server/src/api/controllers/application.js

```diff
@@ -45,11 +44,13 @@ const { getTenantId, isMultiTenant } = require("@budibase/backend-core/tenancy")
 const { syncGlobalUsers } = require("./user")
 const { app: appCache } = require("@budibase/backend-core/cache")
 const { cleanupAutomations } = require("../../automations/utils")
+const context = require("@budibase/backend-core/context")
 
 const URL_REGEX_SLASH = /\/|\\/g
 
 // utility function, need to do away with this
-async function getLayouts(db) {
+async function getLayouts() {
+  const db = context.getAppDB()
```

Comment:
We should probably be consistent with the usage of tenancy context and the other `getAppDB` usages in this pr by importing / using the function directly rather than going through the `context` object

Reply:
in this case I had wanted it to be clear in the application controller when it was getting the DB from context and when it was getting a new/raw one (such as creating an app DB) - for ever other controller this doesn't really matter since they aren't in an app context, so this use case here was specifically to differentiate it from everywhere else it was used.